src/main/java/elysia/ui/InputOutputHandler.java

```diff
@@ -0,0 +1,114 @@
+                Todo newTodo = new Todo(input.substring(5));
+                taskList.addTask(newTodo);
+                output = "Added the task below to your list~\n" + newTodo.toString();
+                output += "Wow! You now have " + taskList.size() + " tasks in your list!";
+            }
+        } else if (input.startsWith("deadline")) {
+            if (input.equals("deadline")) {
+                output = "Hmph! You don't expect me to read your mind for this deadline, do you?";
+            } else {
+                int index = input.indexOf("/");
+                String rawDate = input.substring(index + 4);
+                String date;
+                try {
+                    LocalDate parsedDate = LocalDate.parse(rawDate);
+                    date = parsedDate.format(DateTimeFormatter.ofPattern("MMM d yyyy"));
+                } catch (DateTimeParseException e) {
+                    output = "This doesn't look like a date... I won't be able to do anything with it but as long" +
+                            "as you understand it :)";
+                    date = rawDate;
+                }
+
+                Deadline newDeadline = new Deadline(input.substring(9,index), date);
+                taskList.addTask(newDeadline);
+                output += "Added the task below to your list~\n" + newDeadline.toString();
+                output += "Wow! You now have " + taskList.size() + " tasks in your list!";
+            }
+        } else if (input.startsWith("event")) {
+            if (input.equals("event")) {
+                output = "Hmph! You don't expect me to read your mind for this event, do you?";
+            } else {
+                int index0 = input.indexOf("/");
+                int index1 = input.lastIndexOf("/");
+                Event newEvent = new Event(input.substring(6,index0),
+                        input.substring(index0 + 6, index1),
+                        input.substring(index1+4));
+                taskList.addTask(newEvent);
+                output = "Added the task below to your list~\n" + newEvent.toString();
+                output += "Wow! You now have " + taskList.size() + " tasks in your list!";
+            }
+        } else if (input.startsWith("delete")) {
+            if (input.equals("delete")) {
+                output = "Hmph! What do you even want me to delete?";
+            }   else {
+                int index = Integer.parseInt(input.substring(7));
+                try {
+                    Task deletedTask = taskList.deleteTask(index);
+                    output = "You don't need this task below anymore? Ok deleting it~\n";
+                    output += deletedTask.toString();
+                    output += "Wow! You now have " + taskList.size() + " tasks in your list!";
+                } catch (IndexOutOfBoundsException e) {
+                    output = "Uh oh, this task number does not exist...";
+                }
+            }
+        }
+        else {
```

Comment:
Indent the else statement on the same line as the closing braces of the else if block.